test/api.test.js

```diff
@@ -21,6 +22,32 @@ describe('api', () => {
     expect(typeof RO.api).toBe('object');
   });
 
+  it('should handle 5XX responses', () => {
+    jest.spyOn(auth, 'getToken').mockImplementationOnce((_, callback) => callback(null, 'testToken'));
+
+    nock(RO.urls.getApiBaseUrl())
+      .get('/timeout-error')
+      .once()
+      .reply(504);
+    return new Promise(done => {
```

Comment:
Nit/optional: Add a blank line here for readability?
```suggestion
      .reply(504);
      
    return new Promise(done => {
```